spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-sink-kafka/pom.xml

```diff
@@ -27,9 +28,20 @@
 			<version>2.11.3-SNAPSHOT</version>
 			<type>pom</type>
 		</dependency>
+		<dependency>
+			<groupId>org.springframework.kafka</groupId>
+			<artifactId>spring-kafka</artifactId>
+			<version>${spring-kafka.version}</version>
+		</dependency>
 		<dependency>
 			<groupId>org.springframework.cloud</groupId>
 			<artifactId>spring-cloud-stream-binder-kafka</artifactId>
+			<exclusions>
```

Comment:
This should be managed via spring-cloud-dependencies and as such should only need to set the version number.